crates/biome_js_analyze/tests/specs/nursery/useTrimStartEnd/valid.js

```diff
@@ -0,0 +1,26 @@
+const foo = bar.trimStart();
+const foo = bar.trimEnd();
+bar.trimStart?.()
+bar['trimStart']()
+foo.trimStart(),
+foo.trimStart?.(),
+foo.trimEnd(),
+// Not `CallExpression`
+new foo.trimLeft();,
```

Comment:
A comma after a semicolon seems illegal, can we change all the cases here to end with semicolons? Just like the invalid test cases.